src/App.js

```diff
@@ -8,7 +8,7 @@ import {
   MapManager,
   OurTable
 } from './components'
-import NavBar from './components/NavBar'
+import { NavBar } from './components/NavBar'
```

Comment:
This doesn't compile because `components/NavBar` doesn't have a named export called `NavBar`. 

What I meant by changing the imports is importing the NavBar component in `components/index.js` and then exporting it as a named export:

```javascript
// components/index.js
...
import NavBar from './NavBar'

export {
  ...,
  NavBar
}
```

Since the default behavior when importing from a directory instead of directly from a file is to look at the directory's `index.js`, doing this will allow you to import NavBar like this in other components:

```javascript
import { NavBar } from './components'
```